### PR TITLE
Fixes some minor issues with the test methodology

### PR DESCRIPTION
**Pull request by frznlogic**
Signed-off-by: Oscar Andreasson <oscar.andreasson@pelagicore.com>

### CODE REVIEW

File: doc/chapters/getting-started/examples/09_setcapabilities.sh [hunk 1]
```diff
@@ -3,6 +3,5 @@ gdbus call --system \
 --object-path /com/pelagicore/SoftwareContainerAgent \
 --method com.pelagicore.SoftwareContainerAgent.SetCapabilities \
 0 \
-"['com.pelagicore.temperatureservice.gettemperature',
-'com.pelagicore.temperatureservice.settemperature']"
+"['com.pelagicore.temperatureservice.settemperature']"
```
Comment (tobsan) [inline doc/chapters/getting-started/examples/09_setcapabilities.sh]: So no getTemperature?

Comment (frznlogic) [reply doc/chapters/getting-started/examples/09_setcapabilities.sh]: Nein!

Seriously though... not sure if it serves any purpose for the example. Do you still want to have it?

Comment (tobsan) [reply doc/chapters/getting-started/examples/09_setcapabilities.sh]: No, not if nobody uses it

Comment (frznlogic) [reply doc/chapters/getting-started/examples/09_setcapabilities.sh]: It's an example only, to show how the SetCapabilities call is used inside the Getting Started chapter. The only thing I can imagine it to be good for is as an example on how to pass multiple strings in an array. I will change it  back to end the discussion ;)

**Pull request merged**